File: FrontEnd/src/app/features/reservations/customers/classes/view-models/customer-browser-storage-vm.ts
import { NationalityBrowserStorageVM } from './nationality-browser-storage-vm'

export interface CustomerBrowserStorageVM {

    // PK
    id: number
    // Navigation
    nationality: NationalityBrowserStorageVM
    // Fields
    description: string
    fullDescription: string
    vatNumber: string
    branch: number
    profession: string
    street: string
    number: string
    postalCode: string
    city: string
    phones: string
    email: string
    isActive: boolean

}

File: API/Features/Reservations/Customers/ViewModels/Browser/CustomerBrowserStorageVM.cs
using API.Features.Reservations.Nationalities;
using API.Infrastructure.Interfaces;

namespace API.Features.Reservations.Customers {

    public class CustomerBrowserStorageVM : IBaseEntity {

        // PK
        public int Id { get; set; }
        // Fields
        public string Abbreviation { get; set; }
        public string Description { get; set; }
        public string VatNumber { get; set; }
        public int Branch { get; set; }
        public string Profession { get; set; }
        public string Street { get; set; }
        public string Number { get; set; }
        public string PostalCode { get; set; }
        public string City { get; set; }
        public string Phones { get; set; }
        public string Email { get; set; }
        public bool IsActive { get; set; }
        // Navigation
        public NationalityAutoCompleteVM Nationality { get; set; }

    }

}